app/customer.py

```diff
@@ -0,0 +1,22 @@
+import dataclasses
+import datetime
+
+
+@dataclasses.dataclass
+class Customer:
+    name: str
+    product_cart: dict
+    location: list
+    money: int
+    car: dict
+
+    def make_purchases(self, other: object) -> None:
```

Comment:
Can you annotate `other` with its real type?

Reply:
if i import class in this module, cycled import error will be